.travis.yml

```diff
@@ -36,6 +38,8 @@ services:
   - postgresql
 
 install:
+  # suport for symfony 4.0
+  - if [ "$DEPENDENCIES" = "beta" ]; then composer config minimum-stability beta; fi;
```

Comment:
This change can be removed as `MIN_STABILITY` is used (and handled) already. There's no reason to introduce `DEPENDENCIES`.